ts/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "tdr-generated-graphql",
-  "version": "1.0.0",
+  "name": "@nationalarchives/tdr-generated-graphql",
+  "version": "1.0.3",
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
```

Comment:
The release job for this package failed on the sonatype stage but the npm version had already been bumped. With this and the fact that I was pushing versions without committing the package.json changes, the version is now up to 1.0.8 so you'll need to change it to 1.0.8 in package.json and run `npm i` again to update the package-lock file and it should be fine.

Reply:
Bumped the version to 1.0.8

Reply:
Cool cheers